test-asset-generator/transformers/types/organisation.js

```diff
@@ -1,13 +1,14 @@
 export default class Organisation {
-    constructor(data) {
+    constructor(data, namespace) {
         this.data = data;
+        this.namespace = namespace;
     }
 
     transform() {
         const data = {
             type: 'ManufacturerRecord',
             data: {
-                Name: this.data.name,
+                Name: `${this.data.name}-${this.namespace}`,
```

Comment:
Same comment on namespace as prefix.

Reply:
The current TDG format has namespaces after the text in certain areas.